core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -278,6 +279,7 @@ object KafkaConfig {
   val ReplicaFetchMaxBytesProp = "replica.fetch.max.bytes"
```

Comment:
It's tempting to do this now while we're still in the KIP window. We'll be less inclined to do it in the future if it requires a new KIP.